source/ice_api.c

```diff
@@ -0,0 +1,1824 @@
+/* Standard includes. */
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+
+/* API includes. */
+#include "ice_api.h"
+
+/* STUN API includes. */
+#include "stun_data_types.h"
+#include "stun_serializer.h"
+#include "stun_deserializer.h"
+
+/*-----------------------------------------------------------------------------------------*/
+
+/* Static Functions. */
+static bool Ice_IsSameIpAddress( StunAttributeAddress_t * pAddr1,
```

Comment:
It will be good if we can have a file like ice_api_private.h where we declare all the static functions which the application need not know.

Reply:
added.